src/app/index.ts

```diff
@@ -1,391 +1,391 @@
+        isManifestOnly: isManifestProject,
+        isExcelFunctionsProject: isExcelFunctionsProject,
+        scriptType: (answerForScriptType !== undefined) ? answerForScriptType.scriptType : undefined
+      };
+
+      if (this.options.ts || this.project.projectType === 'react') {
+        this.project.scriptType = typescript; }
+
+      /* Set folder if to output param  if specified */
+      if (this.options.output != null) {
+        this.project.folder = this.options.output; }
+
+      this.project.projectInternalName = _.kebabCase(this.project.name);
+      this.project.projectDisplayName = this.project.name;
+      this.project.projectId = uuid();
+      if (this.project.projectType === excelCustomFunctions) {
+        this.project.host = 'Excel';
+        this.project.hostInternalName = 'Excel';
+      }
+      else {
+        this.project.hostInternalName = this.project.host;
+      }
+      this.destinationRoot(this.project.folder);
+
+      /* Check to to see if destination folder already exists. If so, we will exit and prompt the user to provide
+      a different project name or output folder */
+      this._exitYoOfficeIfProjectFolderExists();
+
+      let duration = this.project.duration;
+      insight.trackEvent('App_Data', { AppID: this.project.projectId, Host: this.project.host, ProjectType: this.project.projectType, isTypeScript: (this.project.scriptType === typescript).toString() }, { duration });
+    }
+    catch (err) {
+      insight.trackException(new Error('Configuration Error: ' + err));
+    }
+  },
+
+  _copyProjectFiles()
+  {
+    return new Promise((resolve, reject) => {
+      try {
+        let language = this.project.scriptType === typescript ? 'ts' : 'js';
+        const starterCode = generateStarterCode(this.project.host);
+        const templateFills = Object.assign({}, this.project, starterCode);
+        let jsonData = new projectsJsonData(this.templatePath());
+        let projectRepoBranchInfo = jsonData.getProjectRepoAndBranch(this.project.projectType, language);
+
+        this._projectCreationMessage();
+
+        // Copy project template files from project repository (currently only custom functions has its own separate repo)
+        if (projectRepoBranchInfo.repo)
+        {
+          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', (projectRepoBranchInfo.branch) ? projectRepoBranchInfo.branch : 'master'], (err) => {
+            const gitFolder = this.destinationPath() + '/.git';
+            if (fs.existsSync(gitFolder)){
+              helperMethods.deleteFolderRecursively(gitFolder);
+            }
+            return err ? reject(err) : resolve();
+          });
+        }
+        else
+        {
+          /* Copy the manifest */
+          this.fs.copyTpl(this.templatePath(`hosts/${_.toLower(this.project.hostInternalName)}/manifest.xml`), this.destinationPath('manifest.xml'), templateFills);
```

Comment:
We will need to update tests is src/tests/manifest-only-projects.ts and src/test/new-projects.ts so that they only look to confirm that "manifest.xml" exists following project generation. Currently they look for the customized manifest file name

Reply:
thanks for that finding

Reply:
@TCourtneyOwen I've updated the tests and had also the chance to remove a couple of unused local vars by moving from the concatenated manifest name to a `const`